From 220c2659a45dd354270a465c45197970dae5f548 Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@intel.com>
Date: Fri, 5 Aug 2016 17:52:18 +0300
Subject: [PATCH] gtk-play: provide similar behaviour for quit and close

In x86 targets, gtk-play just pause rather than quitting the application
when we click the close button (delete-event). Change the callback function
to get similar behaviour when we click on "Quit" menu option.

Upstream-Status: Accepted

Signed-off-by: Maxin B. John <maxin.john@intel.com>
---
 gtk/gtk-play.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtk-play.c b/gtk/gtk-play.c
index a520bef..16afc6b 100644
--- a/gtk/gtk-play.c
+++ b/gtk/gtk-play.c
@@ -177,7 +177,7 @@ load_from_builder (const gchar * filename, gboolean register_sig_handler,
 static void
 delete_event_cb (GtkWidget * widget, GdkEvent * event, GtkPlay * play)
 {
-  gst_player_stop (play->player);
+  gtk_widget_destroy (GTK_WIDGET (play));
 }
 
 static void
-- 
2.4.0

